pyicloud/services/drive.py

```diff
@@ -53,15 +61,23 @@ def get_file(self, file_id, **kwargs):
             params=file_params,
         )
         if not response.ok:
-            return None
-        url = response.json()["data_token"]["url"]
-        return self.session.get(url, params=self.params, **kwargs)
+            self._raise_error(response.status_code, response.reason)
+        package_token = response.json().get("package_token")
+        data_token = response.json().get("data_token")
+        if data_token and data_token.get("url"):
+            return self.session.get(data_token["url"], params=self.params, **kwargs)
+        elif package_token and package_token.get("url"):
```

Comment:
```suggestion
        if package_token and package_token.get("url"):
```

Reply:
I also removed the `else` after the `elif`